BasicSample/Assets/Shared/Scripts/RuntimeInfo.cs

```diff
@@ -59,6 +69,76 @@ private static string GetDisplayInfo()
             }
         }
 
+        private static string GetTrackingInfo()
```

Comment:
nit: is this function better named "GetTrackingStates"?

Reply:
Done